src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -653,5 +654,72 @@ public static IEnumerable<object[]> PostServersStreamsAndExpectedData
         }
 
         #endregion
+
+        #region Delete Method Tests
+
+        [Theory, MemberData("DeleteServers")]
+        public async Task DeleteAsync_Success(Uri deleteUri)
+        {
+            using (HttpClient client = new HttpClient())
+            {
+                HttpResponseMessage response = await client.DeleteAsync(deleteUri);
+                Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            }
+        }
+
+        [Theory, MemberData("DeleteServers")]
+        public async Task SendAsync_HttpDeleteMethodWithContent_Success(Uri deleteUri)
+        {
+            string value1 = Guid.NewGuid().ToString("N");
+            string value2 = Guid.NewGuid().ToString("N");
+            Dictionary<string, string> values = new Dictionary<string, string>{ {"key1", value1}, {"key2", value2} };
+            using (HttpClient client = new HttpClient())
+            {
+                HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Delete, deleteUri);
+                request.Content = new FormUrlEncodedContent(values);
+                HttpResponseMessage response = await client.SendAsync(request);
+                string responseContent = await response.Content.ReadAsStringAsync();
+                foreach(KeyValuePair<string, string> kvPair in values)
+                {
+                    Assert.True(JsonMessageContainsKeyValue(responseContent, kvPair.Key, kvPair.Value));
+                }
+                _output.WriteLine(responseContent);
+            }
+        }
+
+        #endregion
+
+        #region Options Method Tests
```

Comment:
Should we have tests for TRACE, too?  I think we have them now for all of the other standard ones besides that, right?